sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -435,5 +489,68 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
       ret
     }
   }
+
+  override def processLine(line: String, allowInterrupting: Boolean): Int = {
+    var oldSignal: SignalHandler = null
+    var interruptSignal: Signal = null
+    if (allowInterrupting) { // Remember all threads that were running at the time we started line processing.
```

Comment:
line is longer than 100 chars, may fails scalastyle check when building.

Reply:
Got it, thanks.